rest_framework_simplify/views.py

```diff
@@ -233,6 +234,21 @@ def get(self, request, pk=None, parent_resource=None, parent_pk=None):
                     elif type(include_field_tree[1]) in (ForeignKey, ManyToManyRel, ManyToOneRel, OneToOneRel):
                         obj = obj.prefetch_related(include_field_tree[0])
 
+        if len(fields) > 0:
+            for field_name in fields:
+                try:
+                    field = self.model._meta.get_field(field_name)
+                    if hasattr(field, 'multiple') and field.multiple:
+                        multi_field.append(field.name)
+                    if hasattr(field, 'many_to_many') and field.many_to_many:
+                        multi_field.append(field.name)
+                except:
+                    pass
+
+            model_primary_key_name = [field.attname for field in self.model._meta.get_fields() if hasattr(field, 'primary_key') and field.primary_key][0]
+            if model_primary_key_name not in fields and len(multi_field) > 0:
+                simple = False
```

Comment:
I thought this needed to be set to False before we did the prefetchrelated or did you make other changes to address this?